frontend/src/@components/user/UserSearchForm/index.tsx

```diff
@@ -71,19 +70,28 @@ const UserSearchResult = (props: UserSearchResultProps) => {
   }
 
   return (
-    <>
-      {searchedUserList.map(user => (
-        <Styled.SearchedUser
-          key={user.id}
-          isSelected={currentReceiverList.some(receiver => receiver.id === user.id)}
-          onClick={onSelectReceiver(user)}
-        >
-          <Styled.ProfileImage src={user.imageUrl} width='24' alt='프사' />
-          <span>{user.nickname}&nbsp;</span>
-          <Styled.Email>({user.email})</Styled.Email>
-        </Styled.SearchedUser>
-      ))}
-    </>
+    <Styled.SearchedUserContainer>
+      {searchedUserList.map(user => {
+        const isSelected = currentReceiverList.some(receiver => receiver.id === user.id);
+
+        const ariaLabel = isSelected
+          ? '이미 선택되어 있는 유저입니다. 클릭하여 선택을 해제할 수 있습니다.'
+          : '선택되지 않은 유저입니다. 클릭하여 선택할 수 있습니다.';
+
```

Comment:
누가 선택되었는지 알수가 없네요! 이름이 추가되면 좋을것 같고, 아래와 같이 심플하게 가는건 어때요?

```suggestion
        const ariaLabel = `${user.nickname} ${isSelected ? '선택' : '선택해제'}`
```

Reply:
오 !!! 맞네요 직접들어보니깐 ㅋㅋㅋ 말이너무 길어서 짜증이 나는..? 느낌? 사용자들도 불편하다고 느낄 것 같아요!! 또 닉네임을 읽어주지 않으니 알 수가 없겠군요 !! 반영해두었습니다!

Reply:
굿입니다!